lib/Sema/CSSimplify.cpp

```diff
@@ -7631,14 +7783,14 @@ static bool isForKeyPathSubscript(ConstraintSystem &cs,
   return false;
 }
 
-static bool isForKeyPathSubscriptWithoutLabel(ConstraintSystem &cs,
-                                              ConstraintLocator *locator) {
+static bool mayBeForKeyPathSubscriptWithoutLabel(ConstraintSystem &cs,
```

Comment:
Nit: lower-case `b` here.

Reply:
I think the capital `b` is correct here as it’s “may be for key path subscript without label” where “be” is a verb.

Reply:
Okay, makes sense, `<locator> may be for ...` definitely makes sense.